articles/routing-reference.mdx

```diff
@@ -0,0 +1,354 @@
+
+    Values can be a plain text string or REGEX (implemented by [re2](https://github.com/google/re2/wiki/Syntax)). URLs must always start with a `/` character. The REGEX functions `^` and `$` are implicit for each value of this key.
+
+    This key has no default.
+
+    <comp.Example>
+      In this example, we want to redirect any users requesting the "/documentation/articles/choosing-your-ssg/" URL.
+
+      <comp.MultiCodeBlock language="JSON" source="routing.json">
+      ``````````
+      {
+        "routes": [
+          {
+            "from": "/documentation/articles/choosing-your-ssg/",
+            "to": "/documentation/articles/select-your-ssg/",
+            "status": 301
+          }
+        ]
+      }
+      ``````````
+      </comp.MultiCodeBlock>
+    </comp.Example>
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="to" type_markdown="String">
+    This key defines which URL this redirect rule should send visitors to. You must define this key for the redirect rule to function.
+
+    Values can be a plain text string or REGEX (implemented by [re2](https://github.com/google/re2/wiki/Syntax)). URLs must always start with a `/` character. The REGEX functions `^` and `$` are implicit for each value of this key.
+
+    This key has no default.
+
+    <comp.Example>
+      In this example, we want to permanently redirect any users requesting the "/documentation/articles/choosing-your-ssg/" URL to "/documentation/articles/select-your-ssg/".
+
+      <comp.MultiCodeBlock language="JSON" source="routing.json">
+      ``````````
+      {
+        "routes": [
+          {
+            "from": "/documentation/articles/choosing-your-ssg/",
+            "to": "/documentation/articles/select-your-ssg/",
+            "status": 301
+          }
+        ]
+      }
+      ``````````
+      </comp.MultiCodeBlock>
+    </comp.Example>
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="status" type_markdown="Number">
+    This key defines the [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) for a redirect rule.
+
+    Values can be one of the following: `200`, `301`, `302`, `307`, `308`, `404`, or `410`.
+
+    <comp.Notice info_type="info">
+      When using 200, 404, and 410 status codes, `routes.to` must refer to a path on the same Site as `routes.from`, with the domain and protocol excluded.
+    </comp.Notice>
+
+    This key has no default.
+
+    <comp.Example>
+      In this example, we use the HTTP status `301` to indicate a permanent URL change.
+
+      <comp.MultiCodeBlock language="JSON" source="routing.json">
+      ``````````
+      {
+        "routes": [
+          {
+            "from": "/documentation/articles/choosing-your-ssg/",
+            "to": "/documentation/articles/select-your-ssg/",
+            "status": 301
+          }
+        ]
+      }
+      ``````````
+      </comp.MultiCodeBlock>
+    </comp.Example>
```

Comment:
```suggestion
    <comp.Example>
      In this example, we use the HTTP status `308` to indicate a permanent URL change.

      <comp.MultiCodeBlock language="JSON" source="routing.json">
      ``````````
      {
        "routes": [
          {
            "from": "/documentation/articles/choosing-your-ssg/",
            "to": "/documentation/articles/select-your-ssg/",
            "status": 308
          }
        ]
      }
      ``````````
      </comp.MultiCodeBlock>
    </comp.Example>
```